<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>789</width>
    <height>637</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>789</width>
    <height>637</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>789</width>
    <height>637</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Menu</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>789</width>
     <height>637</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>789</width>
     <height>637</height>
    </size>
   </property>
   <widget class="QWidget" name="progress_progressBar" native="true">
    <property name="geometry">
     <rect>
      <x>-50</x>
      <y>0</y>
      <width>881</width>
      <height>721</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>881</width>
      <height>721</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>881</width>
      <height>721</height>
     </size>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="toolTipDuration">
     <number>0</number>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 85, 255);</string>
    </property>
    <widget class="QLabel" name="mainmenu_label">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>40</y>
       <width>301</width>
       <height>54</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 25pt &quot;Berlin Sans FB&quot;;

color: #ffffff;</string>
     </property>
     <property name="text">
      <string>Main Menu</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="playbutton">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>420</y>
       <width>231</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border-radius:20px;font: 20pt &quot;Berlin Sans FB&quot;;
background-color: rgb(0, 0, 127);
color :rgb(255, 255, 255)
}
QPushButton:hover{
background-color: rgb(255, 255, 255);
color :rgb(0, 0, 127)
}
QPushButton:pressed{
background-color: rgb(117,213,192);
color :rgb(0, 0, 127)
}
</string>
     </property>
     <property name="text">
      <string>Play</string>
     </property>
    </widget>
    <widget class="QLabel" name="username_label">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>130</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 25pt &quot;Berlin Sans FB&quot;;

color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>User Name</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="quitbutton">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>420</y>
       <width>231</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border-radius:20px;font: 20pt &quot;Berlin Sans FB&quot;;
background-color: rgb(0, 0, 127);
color :rgb(255, 255, 255)
}
QPushButton:hover{
background-color: rgb(255, 255, 255);
color :rgb(0, 0, 127)
}
QPushButton:pressed{
background-color: rgb(117,213,192);
color :rgb(0, 0, 127)
}</string>
     </property>
     <property name="text">
      <string>Quit</string>
     </property>
    </widget>
    <widget class="QLabel" name="level_label">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>300</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 24pt &quot;Berlin Sans FB&quot;;
background-color: rgb(255, 255, 255);
color :rgb(0, 0, 127)</string>
     </property>
     <property name="text">
      <string>Level :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="resetbutton">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>300</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Berlin Sans FB&quot;;
background-color: rgb(255, 255, 255);
color :rgb(0, 0, 127)</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>reset.png</normaloff>reset.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="playbutton_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border-radius:20px;font: 20pt &quot;Berlin Sans FB&quot;;
background-color: rgb(0, 0, 127);
color :rgb(255, 255, 255)
}
QPushButton:hover{
background-color: rgb(255, 255, 255);
color :rgb(0, 0, 127)
}
QPushButton:pressed{
background-color: rgb(117,213,192);
color :rgb(0, 0, 127)
}</string>
     </property>
     <property name="text">
      <string>Statistics</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar_menu_3">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>210</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;color :rgb(0, 180, 180)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QProgressBar {
     border: 2px solid grey;
     border-radius: 5px;
     background-color:#ffffff;
 }

 QProgressBar::chunk {
     background-color: #ffdb85;
     width: 20px;
 }
</string>
     </property>
     <property name="value">
      <number>60</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="format">
      <string>%p%</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addlevelbutton">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>50</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border-radius:20px;font: 20pt &quot;Berlin Sans FB&quot;;
background-color: rgb(0, 0, 127);
color :rgb(255, 255, 255)
}
QPushButton:hover{
background-color: rgb(255, 255, 255);
color :rgb(0, 0, 127)
}
QPushButton:pressed{
background-color: rgb(117,213,192);
color :rgb(0, 0, 127)
}</string>
     </property>
     <property name="text">
      <string>+Add Level</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>300</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Berlin Sans FB</family>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 18pt &quot;Berlin Sans FB&quot;;
background-color: rgb(255, 255, 255);
color :rgb(0, 0, 127)</string>
     </property>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="level_label_2">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>300</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 19pt &quot;Berlin Sans FB&quot;;
background-color: rgb(255, 255, 255);
color :rgb(0, 0, 127)</string>
     </property>
     <property name="text">
      <string>Select Level :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
